tests/test_pdf_cb.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+import numpy as np
+from zfit.models.physics import CrystalBall, DoubleCB
+import zfit
+from scipy.stats import crystalball
+
+mu = 0.0
+sigma = 0.5
+
+alphal = 1.0
+nl = 2.0
+
+alphar = 3.0
+nr = 4.0
+
+bounds = (-10, 3.0)
+lbounds = (bounds[0], mu)
+rbounds = (mu, bounds[1])
+
+
+def sample_testing(pdf):
+    sample = pdf.sample(n=1000)
+    sample_np = zfit.run(sample)
+    assert not any(np.isnan(sample_np))
+
+
+def eval_testing(pdf, x):
+    probs = zfit.run(pdf.pdf(x))
+    assert not any(np.isnan(probs))
+    return probs
+
+
+def test_cb_dcb():
+
```

Comment:
Add here the line
`zfit.run.create_session(reset_graph=True)`

this resets the graph and session to make sure the test is independent